src/main/java/AddCommand.java

```diff
@@ -0,0 +1,18 @@
+public class AddCommand extends Command {
+    private final Task task;
+
+    public AddCommand(Task task) {
+        this.task = task;
+    }
+
+    @Override
+    public void execute(TaskManager taskManager, Ui ui) {
```

Comment:
I noticed that all the classes that inherited from the Command class did not include Javadocs, especially for the public methods like the execute method. Your method name is quite intuitive but perhaps you could include Javadocs for better clarity?